plugin/session_view.py

```diff
@@ -49,6 +49,8 @@ def __init__(self, listener: AbstractViewListener, session: Session) -> None:
         self.session_buffer = session_buffer
         session.register_session_view_async(self)
         session.config.set_view_status(self.view, "")
+        for status_key, message in session._status_messages.items():
+            self.view.set_status(status_key, message)
```

Comment:
Wouldn't it work better if this was done in `Session.register_session_view_async`?

Reply:
I think this is a bit asymmetric, you're also accessing "private" variables in `session` here.